src/libPMacc/include/types.h

```diff
@@ -44,13 +44,11 @@ namespace PMacc
     using AlpakaSize = size_t;
     using AlpakaIdxSize = int;
 
-#if !defined(__CUDACC__)
-    #define PMACC_ACC_CPU
-#endif
-
+    using AlpakaHost = alpaka::dev::DevCpu;
```

Comment:
Could you rebase this commit and not remove the device detection? I had already corrected this in between.